mediator/med_phases_prep_wav_mod.F90

```diff
@@ -4,6 +4,18 @@ module med_phases_prep_wav_mod
   ! Mediator phases for preparing wav export from mediator
   !-----------------------------------------------------------------------------
 
+  use med_kind_mod          , only : CX=>SHR_KIND_CX, CS=>SHR_KIND_CS, CL=>SHR_KIND_CL, R8=>SHR_KIND_R8
```

Comment:
these don't need to be globals

Reply:
I disagree.  I believe that the kind references should be global since they only then need to be defined in one place.